### PR TITLE
docs(example): add remix-auth-supabase oauth github demo

### PR DESCRIPTION
**Pull request by rphlmr**
Add remix-auth-supabase sign in with github example

### CODE REVIEW

File: examples/remix-auth-supabase/oauth-github/app/root.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      <body>
+        <Outlet />
+        <ScrollRestoration />
+        <script
+          dangerouslySetInnerHTML={{
+            __html: `window.env = ${JSON.stringify(env)}`
+          }}
+        />
+        <Scripts />
+        {process.env.NODE_ENV === "development" && <LiveReload />}
```
Comment (MichaelDeBoey) [inline examples/remix-auth-supabase/oauth-github/app/root.tsx:41]:
```suggestion
        <LiveReload />
```
See #1352

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Hi there! Looks like there are a number of merge conflicts. Also we want to keep the examples directory flat so please move this example to `examples/remix-auth-supabase-github`. Thanks!

File: examples/remix-auth-supabase/oauth-github/app/root.tsx (hunk 1 above)
Comment (rphlmr) [reply examples/remix-auth-supabase/oauth-github/app/root.tsx]: thanks 👍

**Comment by rphlmr** [comment]
Changes done 😇 Thanks for review

**Review by kentcdodds — APPROVED** [review]
Looks good to me 👍 Thanks!

**Pull request merged**